module-apps/application-settings/ApplicationSettings.cpp

```diff
@@ -251,7 +251,7 @@ namespace app
             auto connectResultMsg = static_cast<::message::bluetooth::ConnectResult *>(msg);
             auto device           = connectResultMsg->getDevice();
 
-            if (connectResultMsg->isSucceed()) {
+            if (connectResultMsg->getResult() == message::bluetooth::ConnectResult::Result::Success) {
```

Comment:
You can use `using` to shorten this long `ConnectResult` namespace.

Reply:
Done!